projects/dsp-ui/src/lib/viewer/views/property-view/property-view.component.html

```diff
@@ -36,10 +36,7 @@
                     </div>
                     <!-- Add button -->
                     <!-- temporary hack to hide add button for an XML value -->
-                    <div *ngIf="prop.guiDef.propertyIndex !== 'http://0.0.0.0:3333/ontology/0001/anything/v2#hasRichtext' &&
-                        addButtonIsVisible &&
-                        prop.guiDef.cardinality !== 1 &&
-                        propID !== prop.propDef.id">
+                    <div *ngIf="addValueIsAllowed(prop) && prop.guiDef.propertyIndex !== 'http://0.0.0.0:3333/ontology/0001/anything/v2#hasRichtext'">
```

Comment:
Why does the condition contain a property IRI from the anything test project?

Reply:
ah I get it: but then this should be more generic than `'http://0.0.0.0:3333/ontology/0001/anything/v2#hasRichtext'`, you could use the gui ele

Reply:
I think we should integrate the XML / CKeditor functionality asap, so this becomes unnecessary

Reply:
I agree, I'll make the small change to use the gui element

Reply:
fixed in 94ab6a8